docs/mine-hnt/validators/wallet.mdx

```diff
@@ -30,6 +30,14 @@ At any time use ```-h``` or ```--help``` to get more help for a command.
 
 ```./helium-wallet create basic --network testnet``` 
 
+#### Create a Wallet in a non-interactive / DevOps way
+
+```
+export HELIUM_WALLET_PASSWORD="My Password"
```

Comment:
```suggestion

```

Reply:
That works too, although in this one-line version exporting the variable doesn't make a lot of sense (besides wanted a little more of visual clarity), but works for me!